doc/sphinx/source/inspection/index.rst

```diff
@@ -8,10 +8,76 @@ ability to create image formats. When visualization is enabled, a ``dot`` file
 is generated, along with a ``png``. The default can be adjusted by setting the 
 ``NGRAPH_VISUALIZE_TREE_OUTPUT_FORMAT`` flag to another format, like PDF. 
 
-Note: Large graphs are usually not legible with formats like PDF.  
+.. note:: Large graphs are usually not legible with formats like PDF.  
 
-Large graphs may require additional work to get into a human-readable format. We 
-have provided a script to convert the `most common default output`_, nGraph JSON,
+Large graphs may require additional work to get into a human-readable format.
+
+Manual adjustments with Graphviz
```

Comment:
I'm not entirely sure we need to document all of this here. The algorithm here is not something the user has to deal with, it's just what we use internally to generate graph visualizations.

What might make more sense is an explanation of what the `Send` and `Receive` nodes in the generated PDFs mean (those are the "to" and "from" nodes in the comment that this section came from).

Reply:
I saw all the comments on the cpp file and figured the corresponding doc page might benefit from (that) better explanation; many apologies if I mangled anything WIP here, @aprocter

Can you please link to the lines of code you mean, exactly?  Usually, it's easier to have one source file with one doc, but if the source isn't ready to be documented, we should delete anything that can cause confusion.

Reply:
There's nothing mangled, I'm just saying that the comment this came from is mostly meant for people who are hacking on the graph visualization pass itself. For users I think really all we need to document is:

1. you can get `.dot` files using such-and-such environment variables or other doodads;
2. here's how to generate PDFs/PNGs from those `.dot` files; and, maybe
3. sometimes very long edges will get "cut" to make the graph more tractable to render, especially in training graphs; here's an example.

Reply:
Edited.  I'm not sure the new version is as complete as it could be (quite yet, until the GraphML script get approved), but at least it is less confusing.  Any specific ideas to distinguishing when manual backend "tweaking" of the graph visualization vs when to go straight to the script conversion might be useful.  Probably not easy to document, but perhaps useful.

Reply:
The thing is there's no direct tweaking possible (nor, hopefully, needed). You just get what you get from the `VisualizeTree` pass, which is hopefully good enough.